code/game/objects/structures/lavaland/ore_vent.dm

```diff
@@ -284,9 +287,11 @@
 		if(!user_id_card)
 			continue
 		var/point_reward_val = (MINER_POINT_MULTIPLIER * boulder_size) - MINER_POINT_MULTIPLIER // We remove the base value of discovering the vent
-		user_id_card.registered_account.mining_points += point_reward_val
-		user_id_card.registered_account.bank_card_talk("You have been awarded [point_reward_val] mining points for your efforts.")
+		if(user_id_card.registered_account)
```

Comment:
Don't all ID cards have an account? Even just as a placeholder one that can be overriden later?

Reply:
I've no clue just something I noticed while investigating the harddel

Reply:
I saw a lot of other places in code check for account Id so I assumed it's necessary